tiledb/tests/test_pandas_dataframe.py

```diff
@@ -1190,6 +1191,10 @@ def test_write_var_nullable_str(self):
             data[2] = ""
             assert_array_equal(A.multi_index[:]["data"], data)
 
+    @pytest.mark.skip(
```

Comment:
Please use `xfail` so that we are notified when it starts working and we can remove.